examples/jmeter_listener_example.py

```diff
@@ -0,0 +1,68 @@
+from locust import HttpUser, SequentialTaskSet, task, between
+from locust_plugins.jmeter_listener import JmeterListener
+
+
+import json, random, string
+
+class MakePurchase(SequentialTaskSet):
+    
+    def on_start(self):
+        r_s = ''.join(random.choices(string.ascii_lowercase + string.digits, k=32))
```

Comment:
can you simplify the example a bit more? Remove everything that isnt necessary to show how to use the listener.

Reply:
OK, I'll trim it down